### ISSUE TITLE
Feature Request: Silence AudioMetadataWarning logs

### ISSUE THREAD

**Issue by jsetton** [thebigmunch/google-music-scripts#61]
After looking at recent logs from my automated gms upload job, I noticed an enormous amount of `AudioMetadataWarning` being logged. I assume this is due to the switch to using the audio-metadata module which seems to verify each ID3 tags in great detail. However, it seems from I can see that these warning messages don't have any impact on the upload job in the end. Maybe these could be silenced if they don't actually prevent the upload process. On a side note, the fact that there is no filename in the warning makes it hard to associate a message to a given file.

Here is a list of the most common warnings I got:

```
AudioMetadataWarning:
    Ignoring ``USLT``.
    No lyrics found in unsynchronized lyrics frame.

AudioMetadataWarning:
    Ignoring ``TDOR``.
    Timestamp frame values must conform to the ID3v2-compliant subset of ISO 8601.

AudioMetadataWarning:
    Ignoring ``TDRC``.
    Timestamp frame values must conform to the ID3v2-compliant subset of ISO 8601.

AudioMetadataWarning:
    Ignoring ``TBPM``.
    Numeric text frame values must consist only of digits.
```

**Comment by thebigmunch** [comment]
Yes, this was discussed as part of another issue. In fact, I made a change locally in May. I just haven't been working on my hobby projects much or at all since then. But I'm starting to feel the itch again, so the scratch won't be too far behind.
